firmware_DavidBoard/Core/Inc/Controller.h

```diff
@@ -6,6 +6,10 @@
  */
 
 #include "Common.h"
+#include "Actuators.h"
+#include "I2CManager.h"
```

Comment:
Na, no need to add anything to Controller.h. Especially not the #include "Actuators.h" which no longer exists. Include any file you need in Controller.c. Leave this header file untouched.